app/commands/donations/stripe/sync_subscriptions.rb

```diff
@@ -15,15 +16,27 @@ def create_subscriptions!
       user = subscription_user(subscription)
       next unless user
 
-      Donations::Subscription::Create.(user, subscription)
+      Donations::Stripe::CreateSubscription.(user, subscription)
+    end
+  end
+
+  def update_subscriptions!
+    subscription_ids = local_subscription_ids & active_stripe_subscription_ids
+    subscription_ids.each do |subscription_id|
+      subscription = local_subscriptions[subscription_id]
+      stripe_subscription = active_stripe_subscriptions[subscription_id]
```

Comment:
Can you update L14 to also have the variable called `stripe_subscription` just to make the whole file more consistently easy to read pls.